apps/typeform/frontend/src/utils.ts

```diff
@@ -0,0 +1,92 @@
+import get from 'lodash/get';
+import {
+  ContentType,
+  CompatibleFields,
+  Field,
+  EditorInterface,
+  SelectedFields,
+  TypeFormParameters
+} from './typings';
+
+/** Gets the expireTime from local storage to determine if the token is expired */
+export function tokenIsExpired() {
+  const token = window.localStorage.getItem('token') || '';
+  const expires = window.localStorage.getItem('expireTime') || '0';
+
+  return !token || !expires || Date.now() > parseInt(expires, 10);
+}
+
+export function tokenWillExpireSoon() {
+  const expires = window.localStorage.getItem('expireTime') || '0';
+  const _10Minutes = 600000;
+
+  return !expires || parseInt(expires, 10) - Date.now() <= _10Minutes;
+}
+
+export function resetLocalStorage() {
+  window.localStorage.removeItem('token');
+  window.localStorage.removeItem('expireTime');
+}
+
+export function getToken() {
```

Comment:
I would suggest ensuring that every `window.localStorage.getItem('token') || ''` line is replaced by this function. I would also do the same for expire time.

1. You handle the default case here and don't have to repeat everywhere.
2. If we ever change out the storage method, we wouldn't need to touch the rest of the code since it is abstracted away by this function.